andela/eagle-bn-backend
# #169392340 refactor  and rename user controller

#### What does this PR do?
refactor, rename user controller  and add a service file

#### Description of Task to be completed?

- rename user controller to `users.controller.js`
- add user service folder `user.service.js`
- refactor user controller
#### How should this be manually tested?

 clone the repository
- check if  `users.controller.js` and `users.controller.js` exist
- npm run test

#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
[#169392340](https://www.pivotaltracker.com/story/show/169392340)
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
